chia/daemon/keychain_server.py

```diff
@@ -174,9 +174,8 @@ def get_keychain_for_request(self, request: Dict[str, Any]) -> Keychain:
     async def handle_command(self, command: str, data: Dict[str, Any]) -> Dict[str, Any]:
         try:
             if command == "add_private_key":
-                return await self.add_key(
-                    {"mnemonic_or_pk": data.get("mnemonic", None), "label": data.get("label", None), "private": True}
-                )
+                data["private"] = True
+                return await self.add_key(data)
```

Comment:
In order to maintain backward compatibility, you need to add the following line:
```python
data["mnemonic_or_pk"] = data.get("mnemonic", None)
```